data/migrations/20190109230645_recipes.js

```diff
@@ -0,0 +1,13 @@
+
+exports.up = function(knex, Promise) {
+  return knex.schema.createTable('recipes', table => {
+      table.increments();
+      table.string('recipe_name').notNullable().unique();
+      table.integer('dish_id').unsigned();
```

Comment:
Line 6 may not be necessary are you are already setting the dish_id as a foreign key on line 7